templates/vm.tmpl

```diff
@@ -0,0 +1,34 @@
+{{define "VM"}}
+VMUUID: {{.VMUUID}}
+SiteName: {{.SiteName}}
+CloudComputeService: {{.CloudComputeService}}
+MachineName: {{.MachineName}}
+LocalUserId: {{.LocalUserID}}
+LocalGroupId: {{.LocalGroupID}}
+GlobalUserName: {{.GlobalUserName}}
+FQAN: {{.Fqan}}
+Status: {{.Status}}
+StartTime: {{.StartTime}}
+EndTime: {{.EndTime}}
+SuspendDuration: {{.SuspendDuration}}
+WallDuration: {{.WallDuration}}
+CpuDuration: {{.CPUDuration}}
+CpuCount: {{.CPUCount}}
+NetworkType: {{.NetworkType}}
+NetworkInbound: {{.NetworkInbound}}
+NetworkOutbound: {{.NetworkOutbound}}
+PublicIPCount: {{.PublicIPCount}}
+Memory: {{.Memory}}
+Disk: {{.Disk}}
+StorageRecordId: {{.StorageRecordId}}
```

Comment:
Inconsistent naming. 

Name should be the same as `vm_wrapper.go/vmTemplate` defines. Also other attributes like `LocalUserID`, `ImageID`, etc. should be named in the same format. I suggest using `Id` similarly as APEL uses.

Reply:
Yep, great point. I fixed the inconsistency, but now the linter is complaining that `Id` should be `ID` because of naming conventions in Go.

Reply:
Ok, we won't fight with Go conventions, put `ID` to the structure fields and fix it in this template.